alf/drivers/policy_driver.py

```diff
@@ -102,76 +100,7 @@ def add_experience_observer(self, observer: Callable):
         """
         self._exp_observers.append(observer)
 
-    def get_initial_time_step(self):
-        """Returns the initial action_time_step."""
-        time_step = self.env.current_time_step()
-        action = common.zero_tensor_from_nested_spec(self.env.action_spec(),
-                                                     self.env.batch_size)
-        return make_action_time_step(time_step, action)
-
-    def algorithm_step(self, time_step, state, training, greedy_predict):
```

Comment:
why move `algorithm_step` to  module `common` , what are the benefits ?

Reply:
The ActorThread also needs algorithm_step but it's not derived from PolicyDriver.